services/database/app/app.ts

```diff
@@ -1,44 +1,23 @@
 // lib/app.ts
 import express = require('express');
 import Database from "./database";
-import { Author } from "./models/author";
+import dotenv = require('dotenv');
+import author from './author';
+
+dotenv.config();
+const PORT = process.env.DB_PORT || 3000;
```

Comment:
This should moved into a universal constant.ts file where everything that is in the Env is loaded